src/placeos-rest-api/controllers/zones.cr

```diff
@@ -96,7 +96,7 @@ module PlaceOS::Api
     # list the configured zones
     @[AC::Route::GET("/", converters: {tags: ConvertStringArray})]
     def index(
-      @[AC::Param::Info(description: "only return zones who have this zone as a parent", example: "zone-1234")]
+      @[AC::Param::Info(description: "only return zones who have this zone as a parent (supports comma-separated list)", example: "zone-1234,zone-5678")]
       parent_id : String? = nil,
       @[AC::Param::Info(description: "return zones with particular tags", example: "building,level")]
       tags : Array(String)? = nil,
```

Comment:
We should be using the converter helpers to support multiple `parent_id`s

```crystal
@[AC::Route::GET("/", converters: {tags: ConvertStringArray, parent_id: ConvertStringArray})]
def index(
      @[AC::Param::Info(description: "only return zones who have this zone as a parent (supports comma-separated list)", example: "zone-1234,zone-5678")]
      parent_id : Array(String)? = nil,
```

Reply:
refactored that endpoint